languages/go/exercises/concept/conditionals/.docs/after.md

```diff
@@ -1,5 +1,83 @@
 # After
```

Comment:
We've removed the top-level headers (see https://github.com/exercism/v3/blob/master/docs/maintainers/style-guide.md#headers):

```suggestion
```

This also applies to the other documentation files.